dashboard/app/views/pd/workshop_mailer/teacher_enrollment_reminder.html.haml

```diff
@@ -68,7 +68,7 @@
 
   %p
     Don’t forget to fill out
-    = link_to 'this survey', 'https://form.jotform.com/90066184161150'
+    = link_to 'this survey', 'http://studio.code.org/pd/workshop_survey/csf/pre201'
```

Comment:
```suggestion
    = link_to 'this survey', 'https://studio.code.org/pd/workshop_survey/csf/pre201'
```

Reply:
I should have not missed this. Thanks Brad